chore(ddx): review bare metal docs (#4332)
scaleway/docs-content: pages/dedibox-ip-failover/how-to/configure-network-virtual-machine.mdx

<Macro id="requirements" />

- A Dedibox account logged into the [console](https://console.online.net)

> - A [Dedibox dedicated server](/dedibox/how-to/order-a-server/) with a hypervisor installed
> - An ordered [failover IP](/dedibox-ip-failover/concepts/#failover-ips)
> - A generated [virtual MAC address](/dedibox-ip-failover/concepts/#virtual-mac-address)

Author: ```suggestion
- [Created](/dedibox/how-to/order-a-server/) a Dedibox dedicated server and installed a hypervisor on it
- Ordered a [failover IP](/dedibox-ip-failover/concepts/#failover-ips)
- Generated a [virtual MAC address](/dedibox-ip-failover/concepts/#virtual-mac-address)
```

    ```
2. Create a network configuration as follows. Replace `<failover_IP>` with your failover IP address:
    ```yaml
    network:
      version: 2
      ethernets:
        ens18:
          addresses:
            - <failover_IP>/32
          nameservers:
            addresses: [ "51.159.47.28", "51.159.47.26" ]
          routes:
            - to: default
              via: 62.210.0.1
            - to: 62.210.0.1/32
              via: <failover_IP>
              scope: link
    ```
3. Save the file and exit the text editor.
4. Apply the new configuration:
    ```sh
    sudo netplan apply
    ```

## Debian

1. Log into the virtual machine and edit the network configuration file:
    ```sh
    sudo nano /etc/network/interfaces
    ```
2. Configure the network interface as follows. Replace `<failover_IP>` with your failover IP address:
    ```ini
    auto eth0
    iface eth0 inet static
        address <failover_IP>
        netmask 255.255.255.255
        pointopoint 62.210.0.1
        gateway 62.210.0.1
    ```
3. Save the file and exit the text editor.
4. Set the DNS server information:
    ```sh
    sudo nano /etc/resolv.conf
    ```
5. Add the following DNS resolvers:

>     ```ini

Author: ```suggestion
    ```plaintext
```
